webrender/src/platform/macos/font.rs

```diff
@@ -92,9 +94,14 @@ impl FontContext {
     pub fn new() -> FontContext {
         debug!("Test for subpixel AA support: {}", supports_subpixel_aa());
 
+        // Force CG to use sRGB color space to gamma correct.
+        let contrast = 0.0;
+        let gamma = 0.0;
+
         FontContext {
             cg_fonts: HashMap::new(),
             ct_fonts: HashMap::new(),
+            gamma_lut: gamma_lut::GammaLut::new(contrast, gamma, gamma),
```

Comment:
It is by design at the moment, it's just a port from Skia - http://searchfox.org/mozilla-central/source/gfx/skia/skia/src/core/SkPaint.cpp#1498

Also, that bit of code isn't the src/dst gamma. That's after the luminance conversion - http://searchfox.org/mozilla-central/source/gfx/skia/skia/src/core/SkMaskGamma.cpp#79. We don't actually hit that if block.